<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineBar</class>
 <widget class="QWidget" name="LineBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="lineChartWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>25</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <item row="3" column="0" alignment="Qt::AlignLeft|Qt::AlignBottom">
       <widget class="QLabel" name="lblLineChartTotal">
        <property name="text">
         <string notr="true">Total</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" alignment="Qt::AlignVCenter">
       <widget class="QProgressBar" name="lineChartProgress">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" alignment="Qt::AlignRight|Qt::AlignBottom">
       <widget class="QLabel" name="lblLineChartValue">
        <property name="text">
         <string notr="true">Value</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" alignment="Qt::AlignTop">
       <widget class="QLabel" name="lblLineChartTitle">
        <property name="text">
         <string notr="true">Title</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
